src/config/config.js

```diff
@@ -163,14 +169,14 @@ const config = convict({
     format: String,
     sensitive: true,
     env: 'AZURE_CLIENT_SECRET',
-    default: ''
```

Comment:
Rather than `test_value` can we use:

```
 nullable: true,
 default: null
```

Reply:
The `test_value` is what the stub expects, I'll update the description

Reply:
It feels like we are setting up this to work with stubs. Which feels a bit wrong/specific. Especially as stubs can set these values in stubs via environment variables. What happens if your not using stubs?

Reply:
agreed that setting up for the stubs as default 👍